src/api/emails/send-application-submitted-emails/index.test.ts

```diff
@@ -105,45 +97,6 @@ describe('emails/send-email-application-submitted', () => {
       });
     });
 
-    describe('when application owner and business contact emails are the different', () => {
-      test('it should call sendEmail.application.submittedEmail twice', async () => {
-        application.business.businessContactDetail.email = 'test@test.com';
-        expectedContactSendEmailVars.emailAddress = application.business.businessContactDetail.email;
-
-        await sendApplicationSubmittedEmails.send(application, mockXlsxPath);
-
-        expect(applicationSubmittedEmailSpy).toHaveBeenCalledTimes(2);
-        expect(applicationSubmittedEmailSpy).toHaveBeenCalledWith(expectedSendEmailVars);
-        expect(applicationSubmittedEmailSpy).toHaveBeenCalledWith(expectedContactSendEmailVars);
-      });
-
-      test('it should call sendEmail.application.submittedEmail with the correct template ID', async () => {
-        application.business.businessContactDetail.email = 'test@test.com';
-
-        await sendApplicationSubmittedEmails.send(application, mockXlsxPath);
-
-        expect(underwritingTeamEmailSpy).toHaveBeenCalledTimes(1);
-
-        const templateId = getApplicationSubmittedEmailTemplateIds(application).underwritingTeam;
-
-        expect(underwritingTeamEmailSpy).toHaveBeenCalledWith(expectedSendEmailVars, mockXlsxPath, templateId);
-      });
-
-      test('it should call sendEmail.documentsEmail with the correct template ID twice', async () => {
-        application.business.businessContactDetail.email = 'test@test.com';
-        expectedContactSendEmailVars.emailAddress = application.business.businessContactDetail.email;
-
-        await sendApplicationSubmittedEmails.send(application, mockXlsxPath);
-
-        expect(documentsEmailSpy).toHaveBeenCalledTimes(2);
-
-        const templateId = getApplicationSubmittedEmailTemplateIds(application).account;
-
-        expect(documentsEmailSpy).toHaveBeenCalledWith(expectedSendEmailVars, templateId);
-        expect(documentsEmailSpy).toHaveBeenCalledWith(expectedContactSendEmailVars, templateId);
-      });
-    });
```

Comment:
We need to either get this PR merged in first: https://github.com/UK-Export-Finance/exip/pull/1078

Or revert these changes as it's a breaking change